src/feed-worker.js

```diff
@@ -23,6 +22,15 @@ exports.start = function() {
       });
       // We can pass these objects into another queue, For now just printing to the console.
       console.log(processedPosts);
+      return Promise.resolve(processedPosts);
```

Comment:
NOTE: Because you already `await` on line 8 above, and because this is an `async` function, you can skip the `Promise.resolve()` and `Promise.reject()` throughout this function.  An `async` function implicitly returns a `Promise`.  What you've done here will work, but is unnecessary, and the `Promises` will get unwrapped to expose your result as intended.

If you were in a function, and wanted to convert a synchronous call to be a `Promise`, you could do this.  However, if you're going to use both `Promise.resolve()` and `Promise.reject()`, you might was well `return new Promise((resolve, reject) => { /* your code here */ })`.

We should probably clean this up later in another PR, since this has been merged.